### PR TITLE
doc / Avellaneda strategy

### PR DESCRIPTION
**Pull request by JeremyKono**
Avellaneda Market Making strategy docs

### CODE REVIEW

File: content/strategies/avellaneda-market-making.mdx [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+** Prompt: **
+
+<Prompt
+  prompt="What is the amount of [base_asset] per order? (minimum [min_amount])"
+  response=""
+/>
+
+### `parameters_based_on_spread`
+
+The parameter acts as a toggle between beginner(parameters_based_on_spread=True) and expert mode (False). When equal to true, strategy will require min spread, max spread, volatility multiplier, and inventory risk aversion, while if set to False it will only ask for risk_factor, order_book_depth_factor and order_amount_shape_factor.
```
Comment (keithbaum) [inline content/strategies/avellaneda-market-making.mdx:73]: Missing space
beginner (parameters_based_on_spread)

File: content/strategies/avellaneda-market-making.mdx [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+** Prompt: **
+
+<Prompt
+  prompt="Enter the maximum spread allowed from mid-price in percentage (Enter 1 to indicate 1%)"
+  response=""
+/>
+
+### `vol_to_spread_multiplier`
+
+`vol_to_spread_multiplier` will act as a threshold value to override max_spread when volatility is a higher value.
```
Comment (keithbaum) [inline content/strategies/avellaneda-market-making.mdx:106]: Add a sentence "Multiplier should be larger than 1".

File: content/strategies/avellaneda-market-making.mdx [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+For a multiplier of 1 if volatility is bigger than `max_spread`, then `max_spread` will be (1 x volatility).
```
Comment (keithbaum) [inline content/strategies/avellaneda-market-making.mdx:108]: Remove example for "For a multiplier of 1..."
This comes from a last moment change to the code. Thanks

**Comment by JeremyKono** [comment]
Resolved comments on latest commit

**Review by keithbaum — APPROVED** [review]
LGTM

**Pull request merged**